wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -4,16 +4,14 @@
 {% block bodyclass %}page-explorer {% if ordering == 'ord' %}reordering{% endif %}{% endblock %}
 
 {% block content %}
-    <header class="w-header w-header--merged w-header--no-border">
-        <h1 class="visuallyhidden">Explorer</h1>
-
-        {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' %}
-    </header>
-
+    {% page_permissions parent_page as page_perms %}
+    <div class="w-sticky w-top-0 w-z-header">
+        {% include 'wagtailadmin/pages/page_listing_header.html' with title=parent_page.get_admin_display_title page_perms=page_perms %}
+        {% include "wagtailadmin/shared/side_panels.html" %}
+    </div>
     <form id="page-reorder-form">
         {% csrf_token %}
 
-        {% page_permissions parent_page as parent_page_perms %}
         {% include "wagtailadmin/pages/listing/_list_explore.html" with sortable=1 sortable_by_type=1 full_width=1 show_ordering_column=show_ordering_column show_bulk_actions=show_bulk_actions parent_page=parent_page orderable=parent_page_perms.can_reorder_children show_locale_labels=show_locale_labels %}
```

Comment:
Removing that line isn’t enough, we’d either need to rename the other variable (as I’ve suggested above), or change this other line to use `page_perms`.

Reply:
Adopted the above suggestions. It doesn't seem to break anything, which is great!

![Screenshot from 2022-07-16 18-55-57](https://user-images.githubusercontent.com/86092410/179356872-a1a31c79-8a5f-400c-b80e-cbd9028ed4e8.png)